### PR TITLE
Fill default settings

### PR DESCRIPTION
**Pull request by superboyiii**
Close https://github.com/neo-project/neo-node/issues/583

### CODE REVIEW

**Comment by shargon** [comment]
Could you change also

https://github.com/neo-project/neo-node/pull/584/files#diff-0327b59f4dd09986f49db7500602d9a9R88-R91

**Comment by superboyiii** [comment]
@shargon Change for what?

**Comment by shargon** [comment]
Add the default value, if the section exists, but the values don't

**Comment by superboyiii** [comment]
@shargon I've already add the default values in this PR :) Or which param else do you think is need to have default value as well?

File: neo-cli/Settings.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public class StorageSettings
 
         public StorageSettings(IConfigurationSection section)
         {
-            this.Engine = section.GetSection("Engine").Value;
+            this.Engine = section.GetValue("Engine", "LevelDBStore");
```
Comment (erikzhang) [inline neo-cli/Settings.cs:55]: The default value should be `MemoryStore`. Because `LevelDBStore` may be not installed.

Comment (superboyiii) [reply neo-cli/Settings.cs:55]: But due to config.json's default settings, it's using `LevelDBStore` but not `MemoryStore`, or shall  I adjust config.json as well?
https://github.com/neo-project/neo-node/blob/605fccbef4150118b78f112785eda40de7fc336b/neo-cli/config.json#L4

**Comment by shargon** [comment]
>  I've already add the default values in this PR :) Or which param else do you think is need to have default value as well?

Lines 88-91 https://github.com/neo-project/neo-node/pull/584/files#diff-0327b59f4dd09986f49db7500602d9a9R88-R91

**Comment by superboyiii** [comment]
> > I've already add the default values in this PR :) Or which param else do you think is need to have default value as well?
> 
> Lines 88-91 https://github.com/neo-project/neo-node/pull/584/files#diff-0327b59f4dd09986f49db7500602d9a9R88-R91

The initial value of `public bool IsActive { get; }` is always false. So that it will not influence cli running if the params of `UnlockWalletSettings` is null.
image
But add default value is more safe. I'll add it.

**Comment by shargon** [comment]
dotnet format please

**Comment by superboyiii** [comment]
@shargon Could you fix this? I don't know much about it.
image

**Comment by shargon** [comment]
> Could you fix this? I don't know much about it.

Done, re-run and it works

**Pull request merged**